docker-compose.yml

```diff
@@ -65,3 +65,10 @@ services:
         - ./docker_data/elasticsearch/data:/usr/share/elasticsearch/data
     ports:
       - "9200:9200"
+
+volumes:
```

Comment:
I understand the URL change, but why is this necessary? Most of the services don't have volumes and even the ones who do shouldn't need to share them, which is what this option is for.

Reply:
I thought it's necessary to provide access to elasticsearch by the others containers and fix this problem
```shell
org.elasticsearch.bootstrap.StartupException: ElasticsearchException[failed to bind service]; nested: AccessDeniedException[/usr/share/elasticsearch/data/nodes
```

Reply:
The other containers don't need access to Elasticsearch's volume, as they are not writing to it.